components/daikin_s21/daikin_s21_queries.h

```diff
@@ -50,18 +20,18 @@ namespace StateQuery {
   inline constexpr std::string_view OldProtocol{"F8"};
   inline constexpr std::string_view InsideOutsideTemperatures{"F9"};
   // FA
-  // FB
+  inline constexpr std::string_view FB{"FB"};
   inline constexpr std::string_view ModelCode{"FC"};
   inline constexpr std::string_view IRCounter{"FG"};
   inline constexpr std::string_view V2OptionalFeatures{"FK"};
   // FL
   inline constexpr std::string_view PowerConsumption{"FM"};
-  // FN
-  // FP
-  // FQ
-  inline constexpr std::string_view LouvreAngle{"FR"};
-  // FS
-  // FT
+  inline constexpr std::string_view ITELC{"FN"};
+  inline constexpr std::string_view FP{"FP"};
+  inline constexpr std::string_view FQ{"FQ"};
+  inline constexpr std::string_view LouvreAngleSetpoint{"FR"};
+  inline constexpr std::string_view FS{"FS"};
+  inline constexpr std::string_view FT{"FT"};
```

Comment:
This is suspected to be outdoor unit capacity. Please tell me your outdoor unit model and this value (printed in the debugging summary when `debug_protocol` is enabled).